scripts/install/requestrr.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+# Requestrr installation
+# Author: Brett
+# Copyright (C) 2021 Swizzin
+# Licensed under GNU General Public License v3.0 GPL-3 (in short)
+#
+#   You may copy, distribute and modify the software as long as you track
+#   changes/dates in source files. Any modifications to our software
+#   including (via compiler) GPL-licensed code must also be made available
+#   under the GPL along with build & install instructions.
+#shellcheck source="sources/functions/os"
+. /etc/swizzin/sources/functions/os
+
+echo_progress_start "Downloading source files"
+case "$(_os_arch)" in
+    "amd64") dlurl=$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-x64(.*)zip') >> ${log} 2>&1 ;;
+    "armhf") dlurl=$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-arm(.*)zip') >> ${log} 2>&1 ;;
+    "arm64") dlurl=$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-arm64(.*)zip') >> ${log} 2>&1 ;;
+    *)
+        echo_error "Arch not supported"
+        exit 1
+        ;;
+esac
+
+if ! curl "$dlurl" -L -o /tmp/requestrr.zip >> "$log" 2>&1; then
+    echo_error "Download failed, exiting"
+    exit 1
+fi
+echo_progress_done "Source downloaded"
+
+echo_progress_start "Extracting archive"
+unzip -q /tmp/requestrr.zip -d /opt/ >> "$log" 2>&1
+rm /tmp/requestrr.zip
+mv /opt/requestrr* /opt/requestrr
+echo_progress_done "Archive extracted"
+
+touch /install/.requestrr.lock
+
+echo_progress_start "Creating requestrr user and setting permssions"
+useradd --system -d /opt/requestrr/ requestrr
+chown -R requestrr:requestrr /opt/requestrr/
+chmod +x /opt/requestrr/Requestrr.WebApi
```

Comment:
mmmmmm maybe owner only?
```suggestion
chmod u+x /opt/requestrr/Requestrr.WebApi
```